solutions/swb-app/src/environmentTypeRoutes.ts

```diff
@@ -35,32 +32,20 @@ export function setUpEnvTypeRoutes(router: Router, environmentTypeService: Envir
         ListEnvironmentTypesRequestParser,
         req.query
       );
-      const { paginationToken, pageSize } = request;
-
-      if ((paginationToken && typeof paginationToken !== 'string') || (pageSize && Number(pageSize) <= 0)) {
-        res
-          .status(400)
-          .send('Invalid pagination token and/or page size. Please try again with valid inputs.');
-      } else {
-        const envType = await environmentTypeService.listEnvironmentTypes(request);
-        res.send(envType);
-      }
+      const envTypes = await environmentTypeService.listEnvironmentTypes(request);
```

Comment:
no validation on this one before calling the service?

Reply:
Parser is doing all the validations including the valid pagination